qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/utils/IJDTUtils.java

```diff
@@ -85,5 +86,26 @@ public interface IJDTUtils {
 
 	Location toLocation(IJavaElement element) throws JavaModelException;
 
-	//String getJavadoc(IMember member, DocumentFormat documentFormat) throws JavaModelException;
+	/**
+	 * Returns the JDT text edit as an LSP text edit
+	 *
+	 * @param compulationUnit the compilation unit of the file that was edited
+	 * @param jdtTextEdit     the JDT text edit
+	 * @return the JDT text edit as an LSP text edit
+	 */
+	TextDocumentEdit convertEdit(ICompilationUnit compulationUnit, org.eclipse.text.edits.TextEdit jdtTextEdit);
+
+	/**
+	 * Returns an AST that can be used to create a quick fix for the given
+	 * compilation unit.
+	 *
+	 * @param compliationUnit the compilation that a quick fix is being generated
+	 *                        for
+	 * @return an AST that can be used to create a quick fix for the given
+	 *         compilation unit
+	 */
+	CompilationUnit createQuickFixAST(ICompilationUnit compliationUnit);

```

Comment:
createQuickFixAST implementation uses JDT manipulation and not JDT LS, so you can remove this method. It will avoid for @jeffmaury to implement this method.